### ISSUE TITLE
[FEAT] Add missing prefix to UiInput

### ISSUE THREAD

**Issue by sonya0504** [infermedica/component-library#463]
## Is your feature request related to a problem? Please describe.
According to the new design, there is a need to add a `prefix` to `UiInput`

## Describe the solution you'd like
The design from MGP Component Library:
![image](https://github.com/infermedica/component-library/assets/37744371/75fb4b30-1703-4feb-8e6b-1a0a34521636)

The design from MGP:
![image](https://github.com/infermedica/component-library/assets/37744371/357c7573-2ed4-42bc-85cb-f86ad6893f57)

## What are acceptance criteria?
List the acceptance criteria for this task in a form of a list.

## Additional context
All informations are in task MGPFE-44
